### ISSUE TITLE
Windows App Certification Kit Supported API test failed

### ISSUE THREAD

**Issue by the1schwartz** [TheBlueSky/Microsoft.Data.Sqlite.WinRT#1]
I get these errors when running Windows App Certification Kit Windows Universal 8:

```
API sqlite3_bind_blob in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_bind_double in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_bind_int64 in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_bind_null in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_bind_parameter_count in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_bind_parameter_index in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_bind_parameter_name in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_bind_text in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_busy_timeout in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_changes in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_close in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_close_v2 in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_column_blob in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_column_bytes in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_column_count in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_column_decltype in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_column_double in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_column_int64 in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_column_name in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_column_text in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_column_type in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_db_filename in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_enable_load_extension in sqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_errmsg in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_errstr in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_finalize in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_libversion in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_open_v2 in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_prepare_v2 in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_reset in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_step in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
API sqlite3_stmt_readonly in winsqlite3.dll is not supported for this application type. Microsoft.Data.Sqlite.dll calls this API.
```

Hope you can help fix them. Thanks for enabling Microsoft.Data.Sqlite for WinRT.

Kind regards,
Martin

**Comment by TheBlueSky** [comment]
A quick search shows that some people are having similar issue with other SQLite libraries too; yet, many apps are published to Windows Store with SQLite databases. Need to do some research and testing on this, and see how to other libraries are handling it, if they do.

As you can see in the error, it's "complaining" about the way `winsqlite3.dll` API is accessed; however, `winsqlite3.dll` isn't available in Windows 8.1 and Windows Phone 8.1, as far as I know.

Not sure whether this will prevent approving the app or not; could be just an issue with Windows App Certification Kit.

Thanks for reporting; will report back once I have an update.

**Comment by the1schwartz** [comment]
Ok thanks for the quick reply

**Comment by TheBlueSky** [comment]
This should be fixed now in c4fa3283c8ce3da5759c339a74b69c7b6d3efa1c and NuGet package v1.0.1.
